.github/workflows/label_prs.yaml

```diff
@@ -55,17 +71,35 @@ jobs:
           PR: ${{ github.event.pull_request.html_url }}
 
       - id: infrastructure
-        if: steps.filter.outputs.docs == 'true'
+        if: steps.filter.outputs.infrastructure == 'true'
         run: |
           gh pr edit "${PR}" --add-label "infrastructure"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           PR: ${{ github.event.pull_request.html_url }}
 
+      - id: proposal
+        if: steps.filter.outputs.proposal == 'true'
+        run: |
+          gh pr edit "${PR}" --add-label "proposal"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR: ${{ github.event.pull_request.html_url }}
+
       - id: toolchain
         if: steps.filter.outputs.toolchain == 'true'
         run: |
           gh pr edit "${PR}" --add-label "toolchain"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           PR: ${{ github.event.pull_request.html_url }}
+
+      - id: automated
+        if:
+          contains(fromJSON('["CarbonInfraBot", "dependabot"]'),
+          github.event.pull_request.user.login)
+        run: |
+          gh pr edit "${PR}" --add-label "automated"
```

Comment:
Minor, but maybe "bot" for a shorter label name? These will add up when we have multiple labels...

Reply:
Or are there non-bot automated PRs you also plan to use this for? What's the definition of an "automated" PR? If I do something like clang-tidy fixing, is that automated?

Reply:
Happy to go with `bot`. My goal was specifically to only flag fully bot-managed PRs, anything else seems too complicated to reason about.